lametro/views.py

```diff
@@ -726,7 +726,7 @@ def build_form(self, form_kwargs={}):
                               .facet('inferred_status')\
                               .facet('topics')\
                               .facet('legislative_session', sort='index')\
-                              .highlight()
+                              .highlight(**{'hl.fl': 'text,attachment_text'})
```

Comment:
docs re: this weird `kwarg` https://django-haystack.readthedocs.io/en/master/searchqueryset_api.html?highlight=%22hl.fl%22#SearchQuerySet.highlight